sw/device/silicon_creator/lib/drivers/watchdog.h

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_WATCHDOG_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_WATCHDOG_H_
+#include <stdint.h>
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Initialize the watchdog timer.
+ *
+ * @param timeout_ms The timeout, in milliseconds, after which the watchdog
+ *                   will trigger a reset.
```

Comment:
Can you add that setting this param to `0` results in disabling the WD?

Reply:
Done.